src/modules/simulation/gz_bridge/server.config

```diff
@@ -0,0 +1,60 @@
+<server_config>
+  <plugins>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-physics-system"
+            name="gz::sim::systems::Physics">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-user-commands-system"
+            name="gz::sim::systems::UserCommands">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-scene-broadcaster-system"
+            name="gz::sim::systems::SceneBroadcaster">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-contact-system"
+            name="gz::sim::systems::Contact">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-imu-system"
+            name="gz::sim::systems::Imu">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-air-pressure-system"
+            name="gz::sim::systems::AirPressure">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-air-speed-system"
+            name="gz::sim::systems::AirSpeed">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-apply-link-wrench-system"
+            name="gz::sim::systems::ApplyLinkWrench">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-navsat-system"
+            name="gz::sim::systems::NavSat">
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="gz-sim-sensors-system"
+            name="gz::sim::systems::Sensors">
+            <render_engine>ogre2</render_engine>
+    </plugin>
+    <plugin entity_name="*"
+            entity_type="world"
+            filename="libOpticalFlowSystem.so"
```

Comment:
What if libOpticalFlowSystem.so doesn't exist?

Reply:
You get an error message but everything continues to work as expected.
```
[Err] [SystemLoader.cc:92] Failed to load system plugin [libTest.so] : Could not find shared library.
```